contrib/log_correlation/log4j/src/main/java/io/opencensus/contrib/logcorrelation/log4j/OpenCensusTraceContextDataInjector.java

```diff
@@ -190,18 +188,23 @@ public StringMap rawContextData() {
       case NO_SPANS:
         return getContextData();
       case SAMPLED_SPANS:
-        SpanContext spanContext = tracer.getCurrentSpan().getContext();
+        SpanContext spanContext = getCurrentSpanContext();
         if (spanContext.getTraceOptions().isSampled()) {
           return getContextAndTracingData(spanContext);
         } else {
           return getContextData();
         }
       case ALL_SPANS:
-        return getContextAndTracingData(tracer.getCurrentSpan().getContext());
+        return getContextAndTracingData(getCurrentSpanContext());
     }
     throw new AssertionError("Unknown spanSelection: " + spanSelection);
   }
 
+  private static SpanContext getCurrentSpanContext() {
```

Comment:
feel free to blame me if this turns into terrible idea :P

Reply:
:)  I wanted to use the safer design if there is a possibility that Tracing.getTracer() could return the wrong object.  I also didn't realize that the Spring support used the gRPC context key directly, so I want to be consistent with that.